### ISSUE TITLE
cannot import name 'MRVI' from 'scvi.external'

### ISSUE THREAD

**Issue by zy-fang** [scverse/scvi-tools#2806]
Hi,

I am unable to import the name 'MRVI' from 'scvi.external'. I am currently using scvi version 1.1.2. I found that the mrVI tutorial (https://docs.scvi-tools.org/en/latest/tutorials/notebooks/scrna/MrVI_tutorial.html) uses scvi version 1.1.3. However, I cannot install version 1.1.3 as the highest available version is 1.1.2.

zhaoyu

**Comment by martinkim0** [comment]
Hi, thank you for your interest in using `MRVI`. The model is set to be released with v1.2, so the only way to install it currently is from the `main` branch with the following command:
```
pip install "scvi-tools @ git+https://github.com/scverse/scvi-tools"
```
Sorry about that! We're still applying some changes to the code prior to a stable release.

**Comment by zy-fang** [comment]
Thank you for your reply. It works for me.
